laundry/migrations/0002_auto_20210314_1802.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.1.7 on 2021-03-14 22:02
```

Comment:
All the laundry changes should probably go in a different PR for better discoverability